userguide/tutorials/getting_started.adoc

```diff
@@ -363,6 +364,16 @@ kaui.db.password=killbill
 
 And download the current version of the Kaui war from https://search.maven.org/search?q=a:kaui-standalone[Maven Central].
 
+= Getting Started
```

Comment:
@sbrossie I've merged it to see how it renders but I think you need to change this: `=` is reserved for the very top section and there should be at most one per page.

Also, we should have new title (since the page is already named `Getting Started`).

Reply:
@pierre Agreed. Fixed in https://github.com/killbill/killbill-docs/commit/5d2d6124c3e35d5f21f1f1632ccf7033e5d5a586